xbmc/interfaces/json-rpc/schema/methods.json

```diff
@@ -2075,6 +2075,16 @@
     "params": [],
     "returns": "string"
   },
+  "System.InhibitIdleShutdown": {
+    "type": "method",
+    "description": "Toggle the inhibit status of the idle shutdown timer. When enabled the inhibit prevents the system to shutdown on idle.",
```

Comment:
I would change the second sentence to "When enabled it prevents the system from shutting down on idle".